tests/obc/conftest.py

```diff
@@ -25,14 +27,20 @@
 CONTACTS = ["left", "right"]
 
 
+@pytest.fixture(params=X_II_FORMULAS)
+def x_ii_formula(request) -> str:
+    """Returns a NEVP solver."""
+    return request.param
+
+
 @pytest.fixture(params=NEVP_SOLVERS)
 def nevp(request) -> NEVP:
     """Returns a NEVP solver."""
     return request.param
 
 
 @pytest.fixture(params=BLOCK_SIZE, autouse=True)
-def a_xx(request) -> np.ndarray:
+def a_xx(request) -> tuple[np.ndarray, ...]:
```

Comment:
That's very minor but I think this should be xp. in the type-hint

Reply:
Will address this in #47